### PR TITLE
Review branch

### PR DESCRIPTION
**Pull request by jordaninthewind**
Updated review bullet points to finish task 2

### CODE REVIEW

File: src/controllers/controller.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import User from '../models/model';
+
+class UsersController {
+  constructor() {
+    this.users = [];
```
Comment (MurDaD) [inline src/controllers/controller.js]: If you are using classes, you need to move this ti Users Model instance, but it's ok for now - that will be changed in module 3

File: src/controllers/controller.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  createUser(login, password, age) {
+    const newUser = new User(login, password, age);
+    this.users.push(newUser);
+    return newUser;
+  }
+
+  showAllUsers() {
+    return this.users;
```
Comment (MurDaD) [inline src/controllers/controller.js]: When no users - returns `No users found.`, but should return JSON error

File: src/controllers/controller.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import User from '../models/model';
+
+class UsersController {
+  constructor() {
+    this.users = [];
+  }
+
+  createUser(login, password, age) {
+    const newUser = new User(login, password, age);
```
Comment (MurDaD) [inline src/controllers/controller.js]: Can't create user with this payload:
```{
	"login": "Test1212asd3",
	"password": "asdas2e12312",
	"age": 22
}```

Comment (MurDaD) [reply src/controllers/controller.js]: If unable to create a user - should return a JSON error (here and everywhere)

**Comment by MurDaD** [comment]
Response should always be a JSON - you should create a middleware, or try-catch the response from controller

File: src/controllers/controller.js (hunk 1 above)
Comment (jordaninthewind) [reply src/controllers/controller.js]: Noted -- after reviewing the suggested structure, I'm going to rename this as a service, which makes me think about it holding model data differently. Will update in 3.

File: src/controllers/controller.js (hunk 2 above)
Comment (jordaninthewind) [reply src/controllers/controller.js]: Updated status codes and returned JSON

File: src/controllers/controller.js (hunk 3 above)
Comment (jordaninthewind) [reply src/controllers/controller.js]: Updated -- for the first note, are you sending that payload in x-www-form-urlencoded?

I've added JSON error messaging throughout.

**Pull request closed without merging**